ui-testing-core/src/main/java/com/avito/android/test/UITestConfig.kt

```diff
@@ -28,6 +28,10 @@ object UITestConfig {
      */
     var waiterTimeoutMs: Long = TimeUnit.SECONDS.toMillis(2)
 
+    var activityLaunchTimeoutMilliseconds: Long = TimeUnit.SECONDS.toMillis(10)
```

Comment:
```suggestion
    var activityLaunchTimeoutMs: Long = TimeUnit.SECONDS.toMillis(10)
```
to be consistent with existing properties

Reply:
too late..............

Reply:
I've relesed new version already